cumulusci/cli/cci.py

```diff
@@ -1419,3 +1396,75 @@ def flow_run(runtime, flow_name, org, delete_org, debug, o, skip, no_prompt):
                 "Scratch org deletion failed.  Ignoring the error below to complete the flow:"
             )
             click.echo(str(e))
+
+
+@error.command(name="info", help="Outputs the last part of the most recent traceback")
+@click.argument("num_lines", required=False)
+def info(num_lines):
+    if not CCI_LOGFILE_PATH.is_file():
+        click.echo(f"No logfile found at: {CCI_LOGFILE_PATH}")
+        return
+
+    if not num_lines:
+        output_lines = 30
+
+    output = lines_from_traceback(CCI_LOGFILE_PATH.read_text(), output_lines)
+    click.echo(output)
+
+
+def lines_from_traceback(log_content, num_lines):
+    """Returns the the last num_lines of the logfile,
+    or the whole traceback, whichever is shorter. If
+    no stacktrace is found in the logfile, the user is
+    notified.
```

Comment:
💭 : I'd think that if I pass `num_lines` I should get the last _n_ lines, in addition to the traceback.

Reply:
what if the traceback is 300 lines long?

Reply:
Okay, in that case... maybe `num_lines` is really `max_lines`?